src/assets/locales/de/main.json

```diff
@@ -1066,6 +1066,18 @@
       "metadata_url_required_error": "Metadaten-URL ist erforderlich",
       "metadata_url_invalid_error": "Bitte geben Sie eine gültige Metadaten-URL ein"
     },
+    "clearinghouseSelfDescription": {
+      "heading": "Status des Selbstbeschreibungsdokuments",
```

Comment:
Hi @MaximilianHauer wouldn't be the proper term "Eigenerklärung"

Reply:
Changes Done